caffe2/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(Caffe2_GPU_SRCS)
 # files for CPU and GPU tests respectively.
 set(Caffe2_CPU_TEST_SRCS)
 set(Caffe2_GPU_TEST_SRCS)
+set(Caffe2_MOBILE_TEST_SRCS)
```

Comment:
Why do we need `Caffe2_MOBILE_TEST_SRCS`?  IIUC, `Caffe2_GPU_TEST_SRCS` exist because it is built with a different compiler

Reply:
So that we can build only those tests when we build on mobile? I'm not sure, @bwasti what do you think?

Reply:
cc @Yangqing as well

Reply:
yea, we can chuck everything into CPU_TEST_SRCS.  semantically, we could rename them to CC_SRCS and NVCC_SRCS huh?

Reply:
agree, `CC_SRCS` + `NVCC_SRCS` would make more sense